tinkerbell/smee/values.yaml

```diff
@@ -91,7 +91,7 @@ http:
     port: 8080
     path: ""
   # Additional kernel arguments to pass to the OSIE. (k=v k=v) that are appended to the kernel cmdline in the iPXE script
-  additionlKernelArgs: []
+  additionalKernelArgs: []
```

Comment:
Do we want to simplify this to `extraKernelArgs`? I'm not sure why I used 'additional'.